packages/coil-extension/src/background/services/Stream.ts

```diff
@@ -168,18 +166,29 @@ export class Stream extends EventEmitter {
       return
     }
 
-    // reset our timer when we start streaming.
-    const bandwidth = new AdaptiveBandwidth(
-      this._initiatingUrl,
-      this._tiers,
-      this._debug
-    )
+    const initialSendSeconds = 5
+    const payScheduler = this.container.get(PaymentScheduler)
+    const bandwidth = new MaxBandwidth(0)
+    setTimeout(() => {
+      this._debug('finishing first payment')
+      bandwidth.sendMax = 2 ** 64
+    }, 60_000)
 
     while (this._active) {
       let btpToken: string | undefined
       let plugin, attempt
+
       try {
-        btpToken = await this._anonTokens.getToken(this._authToken)
+        const redeemedToken = await this._anonTokens.getToken(this._authToken)
+        btpToken = redeemedToken.btpToken
+        // On page load, only send a small piece of the first token. After 1 minute, switch to full tokens.
```

Comment:
will this be called on page load, or every time the page becomes active (i.e. switching back into the tab?)

Reply:
While answering this question I realized that it works wrong right now, and found an unrelated bug. But how it should work (and will, once I fix it): switching back to a tab should start the schedule up right where it left off.